benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-hmrc-nonprod.json

```diff
@@ -0,0 +1,35 @@
+[
```

Comment:
Without going through them all, are all these the same permissions as the DWP response writer?

Reply:
Yep.